kubetest2-gke/deployer/options/up.go

```diff
@@ -32,6 +32,9 @@ type UpOptions struct {
 	Version                 string `desc:"Use a specific GKE version e.g. 1.16.13.gke-400, 'latest' or ''. If --build is specified it will default to building kubernetes from source."`
 	WorkloadIdentityEnabled bool   `flag:"~enable-workload-identity" desc:"Whether enable workload identity for the cluster or not. See the details in https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity."`
 	GCPSSHKeyIgnored        bool   `flag:"~ignore-gcp-ssh-key" desc:"Whether the GCP SSH key should be ignored or not for bringing up the cluster."`
+	WindowsEnabled          bool   `flag:"~enable-windows" desc:"Whether enable Windows node pool in the cluster or not."`
+	WindowsNumNodes         int    `flag:"~windows-num-nodes" desc:"For use with gcloud commands to specify the number of nodes for Windows node pools in the cluster."`
+	WindowsMachineType      string `flag:"~windows-machine-type" desc:"For use with gcloud commands to specify the machine type for Windows node in the cluster."`
```

Comment:
Do we need the new `NumNodes` and `MachineType` ? the existing flags should be reusable here.

Reply:
Windows machine type has special requirements. Not all machine types are valid for windows. On the UI, it will force you to change the machine type, so I make it separate. 

For number of nodes, it's also for windows node pools. It takes more time for windows to come up. Normally I only create one node.

Reply:
do you usually have windows and linux nodes? of only one node pool type each?

Reply:
Linux nodepool is required to create windows nodepool, because some cluster-wide service only runs on Linux (like kube-dns). Basically, you cannot create a windows-node-only cluster.